question_answering/test_question_answering.py

```diff
@@ -35,18 +35,19 @@ def test_question_answering(monkeypatch):
     monkeypatch.setattr(transformers.Pipeline, "__call__", mock_pipeline_call)
     input_path = "./data"
     artifact_path = tempfile.mkdtemp()
-    qa_function = mlrun.import_function("function.yaml")
-    qa_run = qa_function.run(
+    project = mlrun.new_project("qa", context="./")
+    fn = project.set_function("question_answering.py", "answer_questions", kind="job", image="mlrun/mlrun")
+    qa_run = fn.run(
         handler="answer_questions",
         params={
             "model_name": "distilgpt2",
             "data_path": input_path,
-            "text_wrapper": (
+            "text_wrapper": [(
```

Comment:
why the list wrapper? you have to give it in a list now? I thought it goes to the `to_group_list` function

Reply:
no need for this' you are right